### PR TITLE
Laties HW01 submission

### PR DESCRIPTION
**Pull request by tlaties**
In order to get the read me file so that I could edit it in RStudio, I had to fork the repository, clone the document from my username to my local computer using "git clone [with the repo url]" in the terminal, and then open the file in RStudio. That was all relatively simple.

I had more trouble with using the different features in markdown. For example, I had a lot of trouble inserting an image so that it would show up once I pushed the file back to GitHub, since I kept using a path from my local computer. I ended up finding a solution on stack overflow that told me to upload the picture to the folder, and then copy the url from GitHub itself into my markdown file. That worked so that the picture showed up after pushing back to GitHub. Another issue I had was getting the lists and headers to show up as actual lists and headers. I fixed this by making sure I followed the guide's example exactly, adding spaces after the asterisks I included. 

To get the files back into Github, I saved the file, and then used the successive commands "git add .", "git commit -m 'with message'", and "git push" in the terminal. This went pretty smoothly, especially since I ended up doing it several times.

### CODE REVIEW

**Review by golnar-bt — COMMENTED** [review]
It was fun learning about you Taylor ! 
I see at first you struggled with headers as I did. I also kept forgetting to put space between # and my words. However, I did not know how to get rid of my generic header hw01. I learned that from you which makes the biography look way nicer. Good job ! 
Also, similar to you, at first I had problems with my image as well. I thought copying the image to my cfss folder should be enough. I see you ended up having the photo shown within your biography. Another way to do it is just use the photo (instead of URL) from your cfss folder as you were trying to do initially. You just have to remember to commit, pull and push the photo as well and it would work. Although this way it will not show up in the txt but you will be able to click on it and open it.  Overall, great job 👍

**Comment by chloehansb** [comment]
Hi Taylor, 

You did a great job following the assignment instructions and using markdown features! I too had some trouble initially uploading my image and also turned to stack overflow for help. In your write-up its clear that you made great use of the resources available and learned from what you found. I'll make sure to do that next time, since I had some issues with the headers. I appreciate that you removed the titles "HW01" and "repository." This made it look more polished.

**Review by chloehansb — COMMENTED** [review]
Hi Taylor, 

You did a great job following the assignment instructions and using markdown features! I too had some trouble initially uploading my image and also turned to stack overflow for help. In your write-up its clear that you made great use of the resources available and learned from what you found. I'll make sure to do that next time, since I had some issues with the headers. I appreciate that you removed the titles "HW01" and "repository." This made it look more polished.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
-# hw01
-Repository for homework 1
+# Taylor Laties
+
+## Biography
+
+My name is Taylor Laties and I am a fourth year undergraduate. I was born in New Jersey, but consider myself to be from Arizona, since I have lived there for the past 9 years (minus my time at school). Here is a picture of my favorite spot in the state: 
+
+![Antelope Canyon](https://raw.githubusercontent.com/tlaties/hw01/master/lower-antelope2_small.jpg)
```
Comment (nnickels) [inline README.md:7]: Beautiful pic! Good job figuring out a solution for your image by using a URL; your prior problem could be related to where the image was located; even if it's stored on a local path in your computer, a copy of it needs to be in the repo folder, then pushed to Github, and its file path called from the repo folder.

**Review by nnickels — COMMENTED** [review]
Evaluation HW-01 for Taylor Laties
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X     |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |     X     |              |            |
| **Achievement, creativity** |           |       X       |            |
| **Ease of access**          |     X      |              |            |

Remarks:

* Nice job on the first homework - you demonstrated several of the features of markdown.
* In your description of your process you describe how from terminal you committed, pushed and pulled, as well as cloning your forked repo. You should also be able to do this with RStudio, and not using terminal. Open up a new RStudio project, with Version Control and Git, and copy your repo URL there. Then the Git window in RStudio will allow you to push and pull.

**Pull request closed without merging**